samples/link-unfurling-stt/nodejs/views/tab.ejs

```diff
@@ -0,0 +1,33 @@
+<head>
+    <script src="https://statics.teams.cdn.office.net/sdk/v1.10.0/js/MicrosoftTeams.min.js"
+        integrity="sha384-6oUzHUqESdbT3hNPDDZUa/OunUj5SoxuMXNek1Dwe6AmChzqc6EJhjVrJ93DY/Bv"
+        crossorigin="anonymous"></script>
+    <script async defer src="https://teams.microsoft.com/share/launcher.js"></script>
+    <script>
+        $(document).ready(function () {
+            microsoftTeams.initialize();
+        });
+    </script>
+    <style>
+        #tabheader {
+            margin-left: 38rem;
+        }
+		
+        #reportimg {
+            margin-left: 28rem;
+        }
+		
+        #stt-btn {
+            margin-left: 40rem;
+            margin-top: 2rem;
+        }
+    </style>
+</head>
+
+<body class="theme-light">
+    <div class="surface">
+        <h3 id="tabheader"> Link unfurling in STT</h3>
+        <img id="reportimg" src="/Images/report.png" width="500" height="425"/>
+        <div id="stt-btn" class="teams-share-button" data-href="https://profile.botframework.com?hello" data-button-type="medium" data-icon-px-size="160" data-preview="true"></div>
```

Comment:
Can we remove the reference to Bot Framework completely from solution and use the ngrok URL (ApplicationBaseUrl from .env file)?